packages/demo-gatsby/data/author.json

```diff
@@ -1,6 +1,6 @@
 {
-  "firstName": "Nolan",
-  "lastName": "Phillips",
+  "firstName": "Ziv",
+  "lastName": "Kalderon",
```

Comment:
Not relevant to this PR :)

Reply:
Pushed fix for it.. Sorry.. I'm not sure why I did push on it